Resources/views/Collector/icon.svg

```diff
@@ -0,0 +1,4 @@
+<svg version="1.1"xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" height="24" viewBox="0 0 24 24" enable-background="new 0 0 24 24" xml:space="preserve">
```

Comment:
maybe browsers are sloppy about xml when parsing svg. but i guess we
still want to fix it.